.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,64 @@
+name: Launch ACAS and run python tests
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  acas:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout acasclient
+        uses: actions/checkout@v3
+        with:
+          path: acasclient
+      - name: Checkout ACAS
+        uses: actions/checkout@v3
+        with:
+          repository: mcneilco/acas
+          path: acas
+      - name: Set Up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+      - name: Install acasclient and dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ./acasclient
+      - name: Set ACAS_TAG to latest release
```

Comment:
@brianbolt This is a great point, thanks for catching this. I'd like to discuss more whether we should be testing against the default branch vs the latest release. I was aiming for latest release because it is:
- Most stable
- Customer-facing

However if we're adding new APIs and testing them, we need a way to test against the default branch.

I wonder if it's time to create a branching structure on acasclient, or at least create a `release/2022.1.x` branch in acasclient that will run tests against `release/2022.1.x`.

Reply:
Filed https://jira.schrodinger.com/browse/ACAS-343 to follow up on this idea. Going to merge this as-is.